Visualization/AFW_Display_Demo.ipynb

```diff
@@ -0,0 +1,303 @@
+    "\n",
+    "This tutorial made use of the [`LowSurfaceBrightness.ipynb` StackClub notebook](https://nbviewer.jupyter.org/github/LSSTScienceCollaborations/StackClub/blob/rendered/SourceDetection/LowSurfaceBrightness.nbconvert.ipynb) by [Alex Drlica-Wagner](https://github.com/LSSTScienceCollaborations/DMStackClub/issues/new?body=@kadrlica)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "## **Step 0) Import Common Python Libraries**\n",
+    "\n",
+    "The [`matplotlib`](https://matplotlib.org/), [`numpy`](http://www.numpy.org/), and [`astropy`](http://www.astropy.org/) libraries are widely used Python libraries for plotting, scientific computing, and astronomical data analysis. We will use these packages in common ways below, including the `matplotlib.pyplot` plotting sublibrary. We also import the [`warnings` library](https://docs.python.org/2/library/warnings.html) to prevent some routine warning messages from printing to the screen."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#allow for matplotlib to create inline plots in our notebook\n",
+    "%matplotlib inline                   \n",
+    "import numpy as np                   #imports numpy with the alias np\n",
+    "import matplotlib.pyplot as plt      #imports matplotlib.pyplot as plt\n",
+    "import warnings                      #imports the warnings library"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's go ahead and import from `astropy` the image stretch limits from the familiar `zscale()` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.visualization import ZScaleInterval     #This function allows use to use the `zscale()` rescaling limits function familiar from, e.g., DS9, to adjust the image stretch.\n",
+    "zscale = ZScaleInterval()                            #create an alias to the `ZScaleInterval()` function"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And let the kernel know that we're happy not to have some useful warnings printed during this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.simplefilter(\"ignore\", category=FutureWarning)   #prevent some helpful but ancillary warning messages from printing during LSST DM Release v16 calls\n",
```

Comment:
This comment about v16 will go stale - better to just say "prevent some ... during LSST DM calls"

Reply:
Fixed, thanks.